mlfinlab/tests/test_fast_ewma.py

```diff
@@ -0,0 +1,41 @@
+"""
+Tests ewma function from fast_ewma module
+"""
+
+import unittest
+import os
+import numpy as np
+import pandas as pd
+
+from mlfinlab.data_structures.fast_ewma import ewma
+
+
+class TestDataStructures(unittest.TestCase):
+    """
+    Test the various financial data structures:
+    1. Imbalance Dollar bars
+    2. Imbalance Volume bars
+    3. Imbalance Tick bars
+    """
+
+    def setUp(self):
```

Comment:
In python you shouldn't name functions like this `thisIsMyFunctions`. Use underscores to separate the words
P.S. in this case it'd be better to name function `set_tick_data_path` or something. Don't be scared to use long names. You read code more often than write it :)

Reply:
FYO, check the library `pathlib`

Reply:
Hi @roaffix, 

Totally agree. Snake case is the way to do it, however in this case the method SetUp() is a special method used in the python unit testing framework to signal to the unit tests which functionality should be run before each unit test.

Reply:
MB. Missed that. Thanks for a reply :)